pkg/terraform/state/gather.go

```diff
@@ -119,15 +118,11 @@ func (h *handler) getExecutions(ns string, spec v1.StateSpec) (map[string]string
 func (h *handler) getEnvVars(ns string, spec v1.StateSpec) ([]coreV1.EnvVar, bool, error) {
 	result := []coreV1.EnvVar{}
 
-	logrus.Debugf("Pulling Vars from Secrets: %d", len(spec.Variables.EnvSecretNames))
```

Comment:
Why drop all debug logs?

Reply:
hrm, thought i wrote a comment on this... i moved on to not debugging this part any more and it was pretty consistent so i dropped the debug as it was too spammy while testing other things.